### ISSUE TITLE
[1.21.1] support iris 1.8.0+

### ISSUE THREAD

**Issue by coco875** [Redi2Go/PhotonicEngine#93]
iris have been update to 1.8.0+beta1 and more and a lot of mod follow this beta it's possible to have the mod support that version to ?

**Comment by Redi2Go** [comment]
Are there any major features that 1.8.0 provides?

**Comment by coco875** [comment]
it's mainly switch to be multiloader, support of sodium 0.6, and by memory dh require 1.8 now, by overriding the dependancy it make a mixin crash so maybe some rename also. If in the future you want to support neoforge through connector or a port then only 1.8 are avaible

**Comment by Tapacywka** [comment]
Yeah, regarding the 1.7 version, 1.8 is much smoother and more stable, I play on a ROG Ally (with Radeon integrated graphics) and on 1.7 Iris + default BSL profile I had ~70-80fps, now when using 1.8 Iris + high BSL profile I have ~110-130fps. I don't know if this is due to the fact that 0.6 Sodium (which is needed for 1.8 Iris) further optimizes the game, but with shaders the game is much lighter then before. + the previously mentioned DH only works with 1.8 Iris. Would love to see Photonics combined with DH.

**Comment by Goldenfreddy0703** [comment]
Hey there, I just recently discovered this mod and I would love to have iris 1.8.0+ support for this mod. I play on steam deck and I find this mod to be amazing when it comes to raytracing and performance. If we could get 1.8.0+ support, that would be amazing.
Thank you

**Comment by mrcrashmer** [comment]
We get more performance, a new version and bsl, fewer errors on loading, and most importantly, all the new mods for optimization and graphics work only with iris 1.8+. It's a little unclear to me why the developer has not yet given a comment on whether he is going to update his mod for compatibility with 1.8 and if so, when it will be.

**Comment by Redi2Go** [comment]
I'm currently working on Iris 1.8+ and 1.21.4. Iris has rewritten a lot of backend stuff, so it will take some time. As a rough estimate I would say 2-3 weeks maybe? I'm on holiday, also :)

**Comment by Redi2Go** [comment]
Related to #120

**Comment by mrcrashmer** [comment]
Of course, it's just that me and many people don't want to choose between updated mods and yours. You have a unique project, and the choice often falls on your side who is familiar with photonics. Once saw it, just don't want to look at other shaders. 
Have a nice holiday and take your time, we will wait as long as necessary.
 Thanks:)



> I'm currently working on Iris 1.8+ and 1.21.4. Iris has rewritten a lot of backend stuff, so it will take some time. As a rough estimate I would say 2-3 weeks maybe? I'm on holiday, also :)

**Comment by Redi2Go** [comment]
The newest [release ](https://modrinth.com/mod/photonics/version/o9YBSd8A)adds support for 1.8.1
